summingbird-online/src/test/scala/com/twitter/summingbird/online/PlannerSpec.scala

```diff
@@ -94,15 +158,14 @@ class PlannerSpec extends WordSpec {
       .sumByKey(store2)
 
     val planned = Try(OnlinePlan(tail))
-    val path = TopologyPlannerLaws.dumpGraph(tail)
 
     planned match {
-      case Success(graph) => assert(true == true)
+      case Success(graph) => {
+        assert(true)
+      }
       case Failure(error) =>
-        val path = TopologyPlannerLaws.dumpGraph(tail)
         error.printStackTrace
-        println("Dumped failing graph to: " + path)
-        assert(false)
+        assert(false, "Dumped failing graph for ' Must be able to plan user supplied Job A ' to: " + TopologyPlannerLaws.dumpGraph(tail))
```

Comment:
I think there is `fail` for this case. http://www.scalatest.org/user_guide/using_assertions